src/Symfony/Bundle/WebProfilerBundle/Resources/views/Script/Mermaid/Makefile

```diff
@@ -0,0 +1,26 @@
+define diagram-orchestration
+import { diagram as flowchartV2 } from '../diagrams/flowchart/flowDiagram-v2.js';
+import { registerDiagram } from './diagramAPI.js';
+
+let hasLoadedDiagrams = false;
+export const addDiagrams = () => {
+  if (hasLoadedDiagrams) {
+    return;
+  }
+  hasLoadedDiagrams = true;
+  registerDiagram('flowchart-v2', flowchartV2, () => true);
+};
+endef
+
+.PHONY: mermaid-flowchart-v2.min.js
+mermaid-flowchart-v2.min.js: | mermaid-$(if $(tag),$(tag),$(error missing “tag” variable))/node_modules
```

Comment:
Where is the value of the tag provided ? It looks like it is passed as argument, but it would be better to have the version defined in the script itself so that we track which version is embedded in the repo (instead of having to know which version was used by the last update by asking the dev to remember it)

Reply:
The tag would have to be provided when calling `make`: `make tag=v10.9.0` e.g.

Will set its default value to `v10.9.0`.

Reply:
instead of a default value, I would prefer a hardcoded value in the makefile. When wanting to update to a new version, you update the hardcoded version, you run the script, and you commit both changes.

Reply:
Forced `tag` value in https://github.com/symfony/symfony/pull/54424/commits/cb041e3586fe8c5a56d4c33df22600a203bfa511